conda_build_config.yaml

```diff
@@ -94,6 +94,24 @@ gmp:
 # glibc used in ctng compiler builds
 gnu:
   - 2.12.2
+# The basic go-compiler with CGO disabled,
+# It generates fat binaries without libc dependencies
+# The activation scripts will set your CC,CXX and related flags
+# to invalid values.
```

Comment:
ok, this means that it won't compile if you accidentally use this alongside compiler('c'), right? (which activation scripts run first, the go compiler or c compiler...?)

Reply:
That comment might be outdated (see https://github.com/conda-forge/go-activation-feedstock/blob/main/recipe/activate.sh). The activation scripts don't do that. But they do set `CGO_ENABLED` based on the variant. If you try to compile a go project taht depends on cgo (either directly or indirectly through a dependency), the Go compiler will loudly complain if `CGO_ENABLED` is set to 0.

Reply:
will it give an error or just a warning, do you know?

Reply:
I mean presumably in the end it won't compile anyway, I'm guessing, so a loud warning should be fine

Reply:
@bwingconda you could update the comment for correctness?

Reply:
@danpetry It will result in a hard error since it won't know what to do (the files will say to compile stuff, but `CGO_ENABLED=0` will say otherwise.